src/Model/Review/Command/GetList.php

```diff
@@ -82,9 +82,11 @@ public function __construct(
     public function execute(SearchCriteriaInterface $searchCriteria = null): ReviewSearchResultInterface
     {
         /** @var Collection $collection */
-        $collection = $this->reviewCollectionFactory->create();
-        $collection->addStoreData();
-        $collection->addStoreFilter($this->getStoreId());
+        $collection = $this->reviewCollectionFactory->create()
+          ->addStatusFilter(
+            \Magento\Review\Model\Review::STATUS_APPROVED
```

Comment:
I think all reviews should be available by api by default.
We can use search criteria to filter reviews by status. 
If filtering using searchCriteria does not work -> we should make some adjustment to make it work.. or add separated rest api method which will return only approved reviews.